js/background.js

```diff
@@ -100,9 +100,21 @@
         return new textsecure.SyncRequest(textsecure.messaging, messageReceiver);
     };
 
+    Whisper.events.on('shutdown', function() {
```

Comment:
`start-shutdown`as the request event, and `shutdown-complete` as the response event. Makes sense.